packager/packager.cc

```diff
@@ -1214,11 +1220,9 @@ struct PackageInfoFinder {
     }
 };
 
-// Sanity checks package files, mutates arguments to export / export_methods to point to item in namespace,
-// builds up the expression injected into packages that import the package, and codegens the <PackagedMethods>  module.
 template <typename ContextType>
-unique_ptr<PackageInfoImpl> getPackageInfo(ContextType ctx, ast::ParsedFile &package,
-                                           const vector<std::string> &extraPackageFilesDirectoryPrefixes) {
+unique_ptr<PackageInfoImpl> runPackageInfoFinder(ContextType ctx, ast::ParsedFile &package,
+                                                 const vector<std::string> &extraPackageFilesDirectoryPrefixes) {
```

Comment:
This change is unrelated, but we have `getPackageInfo` as a method on `PackageDB` and it was confusing me when I was grepping, so I renamed this one.